### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by shivanshsinghal107**

### CODE REVIEW

File: Submissions/dummygithubusername/Assignment-1/assignment-1.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+	def __init__(self):
+		self.chain = [self.createGenesis()]
+
+	def createGenesis(self):
+		return block(time.ctime(), "Genesis Block", "00000")
+
+	def mineBlock(self, data):
+		# current Time = time.ctime()
+		current time = time.ctime()
```
Comment (johnsoncarl) [inline Submissions/dummygithubusername/Assignment-1/assignment-1.py:27]: i guess it is supposed to a wrong variable

File: Submissions/dummygithubusername/Assignment-1/assignment-1.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+from hashlib import sha256
```
Comment (johnsoncarl) [inline Submissions/dummygithubusername/Assignment-1/assignment-1.py:1]: to use hashlib.sha256 you need to import `hashlib` completely!

import hashlib

File: Submissions/dummygithubusername/Assignment-1/assignment-1.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+class block():
+	def __init__(self, timestamp, data, previousHash = ''):
+		self.timestamp = timestamp
+		self.data = data
+		self.previousHash = previousHash
+		self.hash = self.calculateHash()
+
+	def calculateHash(self):
+		#todo
+		self.hash = hashlib.sha256(self.previousHash + self.data)
```
Comment (johnsoncarl) [inline Submissions/dummygithubusername/Assignment-1/assignment-1.py:13]: things have to be encoded before converting them to hashes.

**Review by johnsoncarl — COMMENTED** [review]
Please make the listed changes.

File: Submissions/dummygithubusername/Assignment-1/assignment-1.py (hunk 1 above)
Comment (shivanshsinghal107) [reply Submissions/dummygithubusername/Assignment-1/assignment-1.py:27]: self.timestamp = time.ctime()
I guess this is it.

File: Submissions/dummygithubusername/Assignment-1/assignment-1.py (hunk 2 above)
Comment (shivanshsinghal107) [reply Submissions/dummygithubusername/Assignment-1/assignment-1.py:1]: import hashlib
corrected.

File: Submissions/dummygithubusername/Assignment-1/assignment-1.py (hunk 3 above)
Comment (shivanshsinghal107) [reply Submissions/dummygithubusername/Assignment-1/assignment-1.py:13]: self.hash = hashlib.sha256( (self.previousHash).encode() + (self.data).encode() )
How about this?

Comment (johnsoncarl) [reply Submissions/dummygithubusername/Assignment-1/assignment-1.py:13]: you haven't added timestamp.

also, you didnt add return digested string.

**Pull request closed without merging**